myalbums/test/test_models.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+from myalbums.models import *
+
+class ArtistModelTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        Artist.objects.create(name_artist='Ly',biography='hello world')
+    def test_name_artist_label(self):
```

Comment:
giữa 2 block code cách ra 1 line trống nha em
```suggestion
        Artist.objects.create(name_artist='Ly',biography='hello world')
        
    def test_name_artist_label(self):
```